### PR TITLE
31 restructure course table

### LINKED ISSUE #31: Restructure Course Table
Reference the discussion had here: [meeting](https://usaskca1-my.sharepoint.com/:w:/r/personal/qay871_usask_ca/_layouts/15/Doc.aspx?sourcedoc=%7BD9F07B11-CE34-4AD4-9BB4-23BF50662BEA%7D&file=Prequisite%20Storage%20Planning%20(3.31.2023).docx&action=default&mobileredirect=true)

### PR DESCRIPTION
**Pull request by fraser-mcleod**
I up unused db files and functions as well as created a new table - courseTable2023 with a POST function and initialize function. 
In the next branch will need to add functions to create 'clean_str',  'marked_str', and dependencies as well as an appropriate **PUT** function in views.

### CODE REVIEW

File: server/api/urls.py [hunk 1]
```diff
@@ -3,23 +3,35 @@
...
+
+# Active Endpoints
+########################################################################################
     path('admin/', admin.site.urls), 
 
+    # POST Endpoints
+    path('postCourse2023', views.postCourse2023),
+
+    # GET Endpoints
+
```
Comment (marmikpatel19) [inline server/api/urls.py:15]: Are we adding these in later as we refactor them?

File: server/api/views.py [hunk 2]
```diff
@@ -10,198 +10,271 @@
...
     """
     if request.method == 'POST':
-        serializer = CourseSerializer(data=request.data)
-        if serializer.is_valid():
-            serializer.save()
-            return Response(serializer.data, status=status.HTTP_201_CREATED)
-        return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
-@api_view(['POST'])
-def postPrerequisite(request):
-    """ Insert prerequisite entry into Prerequisite table
```
Comment (marmikpatel19) [inline server/api/views.py:55]: Can we comment out postPrerequisite and postDependency like the rest of the functions instead of deleting them?

Comment (fraser-mcleod) [reply server/api/views.py:55]: Whoops looked at wrong file originally! I will do that

File: server/api/urls.py (hunk 1 above)
Comment (fraser-mcleod) [reply server/api/urls.py:15]: Yes that was my plan, there were just lots of clutter.

File: server/api/views.py [hunk 3]
```diff
@@ -10,198 +10,271 @@
...
-    for record in dependancy_records:
-        serialized_record = DependencySerializer(record)
-        dependant_course = Course.objects.get(id=serialized_record['course_id_depend'].value) # get course object corresponding to 
-        dependant_serializer = CourseSerializer(dependant_course)
-        dependant_course_code = dependant_serializer["course_code"].value
-        data.append(dependant_course_code)
-    # convert python list into json
-    data_json = json.dumps(data)
-    return Response(data_json)
+# @api_view(['POST'])
```
Comment (fraser-mcleod) [inline server/api/views.py:125]: postPrerequisite and postDependency commented out down here!

**Pull request merged**